meson.build

```diff
@@ -0,0 +1,46 @@
+project('highspy', 'cpp',
+       version: '0.0.13',
+  default_options : ['warning_level=2', 'cpp_std=c++17'])
+
+host_system = host_machine.system()
+
+# Add C++ compiler options
+cpp_args = ['-Wall', '-Wextra', '-Wpedantic']
```

Comment:
Aren't these 3 args just warning_level=3 (currently set in default_options to 2)?

Reply:
Yup, I tend to keep them like this since `warning_level=3` also passes `-fimplicit-none` which breaks a lot of older fortran code. Though here that isn't an issue. I'll remove it in favor of `warning_level=3`.

Reply:
Even in general I guess this could be handled by setting warning_level=2 and only manually passing -Wpedantic.